app/clients/cbc_proxy.py

```diff
@@ -143,11 +168,12 @@ def send_canary(
         self,
         identifier,
     ):
-        self._invoke_lambda(payload={'identifier': identifier})
+        self._invoke_lambda_with_failover(payload={'identifier': identifier})
```

Comment:
does this change even make sense? what does this benefit us over calling `self._invoke_lambda` like before?

Reply:
Good point, will change this one to `_invoke_lambda`